src/hashmap.rs

```diff
@@ -129,7 +129,8 @@ where
     A: Allocator,
 {
     /// The default initial size of the map.
-    const INITIAL_SIZE: usize = 8;
+    /// Make sure it's a multiple of CELLS_IN_USE, so that no floats are needed during resizing
```

Comment:
are -> are not

Reply:
Sorry, meant: "no floats are needed" -> "floats are not needed". It's a little more clear.